test/test_python_dispatch.py

```diff
@@ -466,7 +466,7 @@ def _check_is_functional_variant(self, mutable_op, functional_op, args):
 
         # check rest of functional_result is the mutated args
         mutated_args = [maybe_mutated_arg for maybe_mutated_arg, arg in zip(cloned_args, args)
-                        if not torch.allclose(maybe_mutated_arg, arg)]
+                        if not(maybe_mutated_arg is not None and arg is not None and torch.allclose(maybe_mutated_arg, arg))]
```

Comment:
I'm not sure this is going to pass lint (might need a space between `not` and the parenthesis

Reply:
It passed :)